regression-test/suites/fault_injection_p0/test_skip_index_compaction_fault_injection.groovy

```diff
@@ -0,0 +1,239 @@
+      } while (running)
+    }
+  }
+
+  def get_rowset_count = { tablets ->
+    int rowsetCount = 0
+    for (def tablet in tablets) {
+      def (code, out, err) = curl("GET", tablet.CompactionStatus)
+      logger.info("Show tablets status: code=" + code + ", out=" + out + ", err=" + err)
+      assertEquals(code, 0)
+      def tabletJson = parseJson(out.trim())
+      assert tabletJson.rowsets instanceof List
+      rowsetCount +=((List<String>) tabletJson.rowsets).size()
+    }
+    return rowsetCount
+  }
+
+  def check_config = { String key, String value ->
+    for (String backend_id: backendId_to_backendIP.keySet()) {
+      def (code, out, err) = show_be_config(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id))
+      logger.info("Show config: code=" + code + ", out=" + out + ", err=" + err)
+      assertEquals(code, 0)
+      def configList = parseJson(out.trim())
+      assert configList instanceof List
+      for (Object ele in (List) configList) {
+        assert ele instanceof List<String>
+        if (((List<String>) ele)[0] == key) {
+          assertEquals(value, ((List<String>) ele)[2])
+        }
+      }
+    }
+  }
+
+  def run_test = { tableName -> 
+    sql """ INSERT INTO ${tableName} VALUES (1, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (2, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (3, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (4, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (5, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (6, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (7, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (8, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (9, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+    sql """ INSERT INTO ${tableName} VALUES (10, "40.135.0.0", "GET /images/hm_bg.jpg HTTP/1.0", 1, 2); """
+
+    sql "sync"
+
+    def tablets = sql_return_maparray """ show tablets from ${tableName}; """
+    logger.info("tablets: {}", tablets)
+
+    int replicaNum = 1
+    def dedup_tablets = deduplicate_tablets(tablets)
+    if (dedup_tablets.size() > 0) {
+      replicaNum = Math.round(tablets.size() / dedup_tablets.size())
+      if (replicaNum != 1 && replicaNum != 3) {
+        assert(false)
+      }
+    }
+
+    int rowsetCount = get_rowset_count.call(tablets);
+    assert (rowsetCount == 11 * replicaNum)
+
+    try {
+      // first
+      trigger_full_compaction_on_tablets.call(tablets)
+      wait_full_compaction_done.call(tablets)
+    } finally {
```

Comment:
The first time trigger full compaction will fail. 
Need check rowset count to confirm.